src/bundle/Resources/public/scss/_page-title.scss

```diff
@@ -1,15 +1,35 @@
 .ez-page-title {
     padding-bottom: 1rem;
+    display: grid;
+    grid-template-columns: 1fr 40px;
+    grid-column-gap: 10px;
+    grid-template-rows: auto;
+    grid-template-areas:
+        "content-item bookmark"
+        "content-type-name .";
 
     h1 {
         margin-bottom: 0;
+        padding-left: 2.25rem;
+        text-indent: -2.4rem;
     }
 
     h4 {
-        padding-left: 2.5rem;
+        padding-left: 2.25rem;
         font-size: 1.125rem;
         font-weight: 400;
         color: $ez-color-base-light;
     }
 
-}
+    .ez-page-title__content-item {
+        margin-bottom: 0;
+        grid-area: content-item;
+        justify-self: start;
+    }
+
+    .ez-page-title__content-type-name {
+        grid-area: content-type-name;
+        justify-self: start;
+        align-self: start;
```

Comment:
Unless I'm mistaken, it applies to elements in the flexbox model? You're using grid model.